packages/manager/modules/pci/src/components/project/instance/ssh-keys/ssh-keys.html

```diff
@@ -2,111 +2,116 @@
 
 <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
 
-<div data-ng-if="!$ctrl.loaders.keys">
-    <oui-field
-        data-label="{{:: 'pci_project_instance_ssh_key' | translate }}"
-        data-size="xl"
-        data-ng-if="!$ctrl.addKeyMode"
-        class="oui-input-group"
-    >
-        <div class="d-inline-block">
-            <oui-select
-                class="mr-2 mb-0"
-                data-name="sshKey"
-                data-model="$ctrl.key"
-                data-match="name"
-                placeholder="{{:: 'pci_project_instance_ssh_pick' | translate }}"
-                data-items="$ctrl.availableKeys"
-                data-on-change="$ctrl.selectKey(modelValue)"
-                required
-            >
-                <span data-ng-bind="$item.name"></span>
-            </oui-select>
-        </div>
-        <oui-button
-            class="align-top"
-            data-variant="secondary"
-            data-on-click="$ctrl.displayAddForm()"
-        >
-            <span data-translate="pci_project_instance_ssh_add"></span>
-        </oui-button>
-
-        <div class="oui-field__helper">
-            <p
-                data-ng-if="!$ctrl.loaders.guide"
-                data-translate="pci_project_instance_ssh_key_help_required"
-                data-translate-values="{
-                    guideUrl: $ctrl.guideUrl
-                }"
-            ></p>
-            <p data-translate="pci_project_instance_ssh_key_infos"></p>
-        </div>
-    </oui-field>
-
-    <div data-ng-if="$ctrl.addKeyMode" class="pb-4">
+<form name="$ctrl.sshKeyForm" novalidate>
```

Comment:
Be careful when using `form` tags in components. Make sure this `form` tag is not nested inside another `form` tag by using the component, like so
```html
<form>
   ...
   <ssh-keys />
</form>
```
This can lead to undesired effects
If this is the case, use a `ng-form` tag instead. If not, please ignore this comment